go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1260,6 +1279,25 @@ func (e *Executor) initMigrationSQLMode(ctx context.Context, onlineDDL *schema.O
 	return deferFunc, nil
 }
 
+// initConnectionLockWaitTimeout sets the given lock_wait_timeout for the given connection, with a deferred value restoration function
+func (e *Executor) initConnectionLockWaitTimeout(ctx context.Context, conn *connpool.Conn, lockWaitTimeout time.Duration) (deferFunc func(), err error) {
+	deferFunc = func() {}
```

Comment:
We are using named return params so this shouldn't be needed.

Reply:
It's needed to be not-nil. I'm invoking the resulting function and don't want to worry about it being nit.